content/20.treatments.md

```diff
@@ -166,6 +166,82 @@ A list of current studies and their results, using carefully the information req
+In vitro studies of SARS-CoV-2 infection show both HCQ and CQ to be effective in inhibiting viral replication, with HCQ being more potent [@doi:10.1093/cid/ciaa237].
+Thus, there is excitement about HCQ as a potential therapeutic for SARS-CoV-2 and COVID-19.
+This section will discuss current efforts on assessing the potential role of HCQ in treating COVID-19.
+
+###### Current Evidence
+
+Gautret et al conducted a non-randomized, non-blinded, non-placebo clinical trial on 42 hospitalized patients comparing HCQ to standard care [@doi:10.1016/j.ijantimicag.2020.105949].
+This trial found patients who received HCQ showed higher rates of virological clearance by nasopharyngeal swab on Days 3-6 when compared to standard care.
+This study also treated six patients with both HCQ + azithromycin and found this combination therapy to be more effective than HCQ alone.
+This study showed design and analysis weaknesses that severely limit interpretability of results.
+These weaknesses include: lack of randomization, lack of blinding, lack of placebo, lack of Intention-To-Treat analysis, lack of correction for sequential multiple comparisons, trial arms entirely confounded by hospital, false negatives in outcome measurements, lack of trial pre-registration, and small sample size.
+Two of these issues are due to inappropriate data analysis and can therefore be corrected post-hoc by recalculating p-values (lack of Intention-To-Treat analysis and multiple comparisons.)
+However, all other issues are fundamental design flaws and can not be corrected for.
+Thus, conclusions cannot be generalized outside of the study.
+Nevertheless, this study does provide promising preliminary data on the use of HCQ in COVID-19 treatment and supports further investigation of HCQ in large, rigorous clinical trials.
+
+A randomized, non-placebo trial of 62 COVID-19 patients at the Renmin Hospital of Wuhan University studied whether HCQ decreased time to fever break or time to cough relief when compared to standard care [@doi:10.1101/2020.03.22.20040758].
+This trial found HCQ decreased both time to fever break and time to cough relief.
+However, this study also showed design and analysis weaknesses that disallow general interpretation of results.
+These weaknesses include: lack of placebo, inappropriate choice of outcomes, lack of correction for multiple primary outcomes, lack of sufficient detail to understand analysis, drastic disparities between pre-registration and published protocol, and small sample size.
+It is important to note here that the authors claim "neither the research performers nor the patients were aware of the treatment assignments."
+However, this seems impossible in a non-placebo trial - at the very least, providers would know whether they were giving HCQ or nothing, and this knowledge could lead to systematic differences in how care is given.
```

Comment:
```suggestion
However, this seems impossible in a non-placebo trial - at the very least, providers would know whether they were administering a medication or nothing, and this knowledge can lead to systematic differences in how care is given.
```
I think this might be what you meant -- either they gave a medication or they didn't, regardless of whether they knew it was HCQ!